lang/en.json

```diff
@@ -228,6 +228,19 @@
       "developerTest": "test account",
       "appDeveloper": "dev account",
       "standard": "standard"
+    },
+    "PrivateAppUserTokenManager": {
+      "enabled": "Private app user tokens is enabled for accountId~{{ accountId }}",
+      "disabled": "Private app user tokens is not enabled for accountId~{{ accountId }}, skipping call to fetch for appId~{{ appId }}",
+      "create": "Creating new private app user token for appId~{{ appId }}",
+      "cached": "Using cached private app user token for appId~{{ appId }}",
+      "refresh": "Updating or rereshing private app user token for appId~{{ appId }} ",
+      "refreshScheduled": "Scheduling refresh of token for appId~{{ appId }} expiring at {{ expiresAt }} to run at {{ refreshTime }} ",
+      "errors": {
+        "noScopes": "Unable to use private app user tokens for accountId~{{ accountId }}. Please regenerate the Personal Access Key.",
+        "apiError": "Unable to get or create private app user token for appId~{{ appId }} on accountId~{{ accountId }}. {{ messageDetail }} failed.",
+        "refreshFailed": "Unable to refresh private app user token for appId~{{ appId }} on accountId~{{ accountId }}."
```

Comment:
You can remove the `~`s here and just say `appId {{ appId }}`, we don't use notation like that anywhere else